evergreen.yml

```diff
@@ -1510,26 +1496,14 @@ tasks:
         variant: "*"
       - name: rustfmt
         variant: "*"
-      - name: windows-unit-test
-        variant: "windows-64"
-      - name: windows-integration-test
-        variant: "windows-64"
-      - name: windows-result-set-test
-        variant: "windows-64"
-      - name: ubuntu-unit-test
-        variant: "ubuntu2204"
-      - name: ubuntu-integration-test
-        variant: "ubuntu2204"
-      - name: ubuntu-result-set-test
-        variant: "ubuntu2204"
-      - name: asan-unit-test
+      - name: unit-test
+        variant: "*"
+      - name: integration-test
         variant: "*"
-      - name: asan-integration-test
+      - name: result-set-test
         variant: "*"
```

Comment:
One thing I thought I should mention is that now the `snapshot` task depends on all of the variants passing the `unit_test` and `integration_test` since the `asan` tests need to pass for all variants and I can't break the tests apart if they are in a more general test.